src/scripts/main.js

```diff
@@ -15,14 +15,26 @@ const food = {
       'Banana': {},
       'Pineapple': {},
     },
-
   },
 };
 
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    const li = document.createElement('li');
+
+    li.innerText = key;
+    ul.append(li);
+
+    if (Object.keys(data).length !== 0) {
+      createTree(li, data[key]);
```

Comment:
The created `ul` element should be appended to `li` before recursively calling `createTree`. This way, the nested `ul` will be inside the correct `li`

Reply:
![image](https://user-images.githubusercontent.com/88326325/231983729-bb1ee45c-88fb-4338-b6b8-56d00150b591.png)
That`s what I corected 21 hours ago